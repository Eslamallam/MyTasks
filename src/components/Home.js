import React from "react";
import "../styles/home.css";

const Home = () => {
  return (
    <div className="home">
      <h1>My Tasks</h1>
      <p>
        This is a simple application to manage daily tasks easily, you can
        create new tasks with category or remove existing one.{" "}
      </p>
    </div>
  );
};

export default Home;
