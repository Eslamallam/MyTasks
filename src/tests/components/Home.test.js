import React from "react";
import Adapter from "enzyme-adapter-react-16";
import { shallow, configure } from "enzyme";
import toJSON from "enzyme-to-json";
import Home from "../../components/Home";

configure({ adapter: new Adapter() });

test("should render Home correctly", () => {
  const wrapper = shallow(<Home />);
  expect(wrapper.find("h1").text()).toBe("My Tasks");
  expect(toJSON(wrapper)).toMatchSnapshot();
});
