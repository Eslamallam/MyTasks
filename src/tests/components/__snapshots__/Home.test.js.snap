// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Home correctly 1`] = `
<div
  className="home"
>
  <h1>
    My Tasks
  </h1>
  <p>
    This is a simple application to manage daily tasks easily, you can create new tasks with category or remove existing one.
     
  </p>
</div>
`;
